en/chromebook_setup/instructions.md

```diff
@@ -1,187 +1,35 @@
-```
-mkdir djangogirls
-cd djangogirls
-python3 -m venv myvenv
-source myvenv/bin/activate
-pip install django~={{ book.django_version }}
-```
-
-(note that on the last line we use a tilde followed by an equal sign: `~=`).
-
-### GitHub
-
-Make a [GitHub](https://github.com) account.
-
-### PythonAnywhere
-
-The Django Girls tutorial includes a section on what is called Deployment,
-which is the process of taking the code that powers your new web application
-and moving it to a publicly accessible computer (called a server) so other
-people can see your work.
-
-This part is a little odd when doing the tutorial on a Chromebook since we're
-already using a computer that is on the Internet (as opposed to, say, a laptop).
-However, it's still useful, as we can think of our Cloud 9 workspace as a place
-for our "in progress" work and Python Anywhere as a place to show off our stuff
-as it becomes more complete.
-
-Thus, sign up for a new Python Anywhere account at
-[www.pythonanywhere.com](https://www.pythonanywhere.com).
+For people using a Chromebook or those with limited memory on their laptops, we recommend using a cloud IDE environment. This allows you to interact with the command line, Python, and Django directly through your browser, where a code editor is already installed.
+
+Your installation experience will be a little different.
+
+- Make a [GitHub](https://github.com) account.
+- Create a GitHub project. Visit [this](https://github.com/new) link, choose any name for your project, and make it public (needed for deployment, you can make it private later). Also, add a `README.md` and `.gitignore` file.
+
+![](../chromebook_setup/images/create-new-repo.png)
+
+In case you using this instruction, some steps of the tutorial don't have to be done again ("Deploy/Git" [part](https://tutorial.djangogirls.org/en/deploy/#installing-git)). 
+
+- Start a Codespace. Go to GitHub [Codespaces](https://github.com/codespaces/new) and select the repository you just created. Click "Create Codespace".
+
+![](../chromebook_setup/images/create-new-codespace.png)
+
+Wait a little bit and you'll see something like that:
+
+![](../chromebook_setup/images/codespace-preview.png)
+
+The [VSCode](https://code.visualstudio.com) editor will open for you automatically. If you see a notification that says "Install Python", please click on it. If not prompted, click the "Extensions" icon on the left sidebar in the Codespace editor. Search for "Python" and click "Install."
+
+![](../chromebook_setup/images/vscode-install-python.png)
+
+The bash terminal (similar to Linux) is at the bottom of the page.
+
+![](../chromebook_setup/images/codespace-preview.png)
+
+By default, the GitHub Codespace environment will be deleted after 1 month (this applies only to the environment, not your code). To prevent auto-deletion, you can adjust the settings on the page: https://github.com/codespaces 
+
+![](../chromebook_setup/images/codespace-auto-delete-off.png)
```

Comment:
In case the list is kept, should this be part of the list? (Currently it splits it into separate lists.) If you want to make it part of a previous list item, indent it in the markdown source.

Reply:
I've made it a part of the previous list